doc/acl/Everflow-test-plan.md

```diff
@@ -0,0 +1,1157 @@
+
+- [Overview](#Overview)
+  - [Scope](#Scope)
+  - [Summary of the existing everflow test plan](#Summary-of-the-existing-everflow-test-plan)
+  - [Extend the test plan to cover both ingress and egress mirroring](#Extend-the-test-plan-to-cover-both-ingress-and-egress-mirroring)
+    - [What new enhancements need to be covered?](#What-new-enhancements-need-to-be-covered)
+      - [Egress ACL table](#Egress-ACL-table)
+      - [Egress mirroring](#Egress-mirroring)
+    - [Some existing areas not covered by the existing scripts](#Some-existing-areas-not-covered-by-the-existing-scripts)
+      - [ACL rule for matching "IN_PORTS"](#ACL-rule-for-matching-IN_PORTS)
+      - [IPv6 everflow](#IPv6-everflow)
```

Comment:
Could you add ICMP code/type test as well?

Reply:
@stcheng Does SONiC supports matching ICMP type/code? If yes, could you please point me to relevant document/design/code? I need to know how to add ACL rules for matching ICMP type/code. Thanks!

Reply:
hello, yes https://github.com/Azure/sonic-swss/blob/cadc5bd8542d8691f3ce39f4d2569f786a558ff7/orchagent/aclorch.cpp#L343 although `acl-loader` does not support icmp code/type, if you use configuration database, you could still program it.

Reply:
Thanks for the info! I have updated the test plan to cover ACL rules matching ICMP type and code.